src/sqlancer/common/schema/AbstractTable.java

```diff
@@ -60,6 +60,15 @@ public C getRandomColumn() {
         return Randomly.fromList(columns);
     }
 
+    public C getRandomColumnWithFilter(String typeFilter) {
+        for (C c : columns) {
+            if (c.getType().toString().equals(typeFilter)) {
+                return c;
+            }
+        }
+        return null;
```

Comment:
It might simplify the code that uses this method to throw an `IgnoreMeException` here. To reflect this in the method name, the method could be called `getRandomColumnOrBailout`.